server/portal/settings/settings_secret.example.py

```diff
@@ -289,13 +280,6 @@
 # NOTE: Use the _AGAVE_TENANT_ID URL value when setting up the tracking property.
 _GOOGLE_ANALYTICS_PROPERTY_ID = 'UA-XXXXX-Y'
 
-########################
-# Elasticsearch
-########################
-_ES_HOSTS = 'core_portal_elasticsearch:9200'
-_ES_AUTH = 'username:password'
-_ES_INDEX_PREFIX = 'cep-dev-{}'
-
 ########################
 # WORKBENCH SETTINGS
 ########################
```

Comment:
__Lost Property Check__

Stache has many more. Should Core have them or note them as available in a comment?

```
    …
    "compressApp": 'zippy',
    "extractApp": 'extract',
    …
    "namespace": _PORTAL_NAMESPACE
```

Reply:
Stache is sort of "looking ahead" for some of these. I updated those we know are coming, but these additions to the example file should ideally be included in the original PR